docs/schemas/itemMeta.json

```diff
@@ -0,0 +1,996 @@
+      "id": "http://talentappstore.github.io/tas-tenant-apis/tas-profile",
+      "title": "tas-profile",
+      "description": "A social profile question item",
+      "type": "object",
+      "properties": {
+        "type": {
+          "type": "string",
+          "enum": ["profile"]
+        },
+        "acceptedNetworks": {
+          "description": "List of accepted social networks",
+          "type": "array",
+          "items": { "type": "string" }
+        },
+        "defaultNetwork": {
+          "description": "Default social network",
+          "type": "string"
+        }
+      }
+    },
+
+    "tas-url": {
+      "$schema": "http://json-schema.org/draft-04/schema",
+      "id": "http://talentappstore.github.io/tas-tenant-apis/tas-url",
+      "title": "tas-url",
+      "description": "A url question item",
+      "type": "object",
+      "properties": {
+        "type": {
+          "type": "string",
+          "enum": ["url"]
+        },
+        "maxSize": {
+          "description": "Max number of characters",
+          "type": "number",
+          "default": 400
+        }
+      }
+    },
+
+    "tas-phone": {
+      "$schema": "http://json-schema.org/draft-04/schema",
+      "id": "http://talentappstore.github.io/tas-tenant-apis/tas-phone",
+      "title": "tas-phone",
+      "description": "A phone question item",
+      "type": "object",
+      "properties": {
+        "type": {
+          "type": "string",
+          "enum": ["phone"]
+        },
+        "maxSize": {
+          "description": "Max number of characters",
+          "type": "number",
+          "default": 40
+        },
+        "phoneTypes": {
+          "description": "List of phone types allowed",
+          "type": "array",
+          "items": { "type": "string" }
+        }
+      }
+    },
+
+    "tas-email": {
+      "$schema": "http://json-schema.org/draft-04/schema",
+      "id": "http://talentappstore.github.io/tas-tenant-apis/tas-email",
+      "title": "tas-email",
+      "description": "A email question item",
+      "type": "object",
+      "properties": {
+        "type": {
+          "type": "string",
+          "enum": ["email"]
+        }
+      }
+    },
+
+    "tas-enum": {
+      "$schema": "http://json-schema.org/draft-04/schema",
+      "id": "http://talentappstore.github.io/tas-tenant-apis/tas-enum",
+      "title": "tas-enum",
+      "description": "An single-select question item",
+      "type": "object",
+      "properties": {
+        "type": {
+          "type": "string",
+          "enum": ["enum"]
+        },
+        "options": {
+          "description": "Possible string choices for this item",
+          "type": "array",
+          "items": {
+            "type": "object",
+            "properties": {
+              "id": { "type": "string" },
+              "displayText": { "type": "string" }
+            }
+          }
+        }
+      }
+    },
+
+    "tas-enumMulti": {
```

Comment:
Copy and paste alert! Shouldn't this be merged with the previous LOV type question type? They seem to be virtually identical.